etc/conf/base.py

```diff
@@ -1,3 +1,7 @@
+"""
+This base config script gets automatically executed for all platforms
+via configure if no sub-configuration is specified.
```

Comment:
Actually I think the behavior is that it gets automatically executed in all cases.